Doppler.Currency/Controllers/CurrencyController.cs

```diff
@@ -21,7 +20,7 @@ public class CurrencyController : ControllerBase
         public CurrencyController(ILogger<CurrencyController> logger, ICurrencyService currencyService) => 
             (_logger, _currencyService) = (logger, currencyService);
 
-        [HttpGet("{currencyCode}/{date}")]
+        [HttpGet("billing-currency/{currencyCode}/{date}")]
```

Comment:
Willthe complete URL in production be something like `https://apis.fromdoppler.com/currency/billing-currency/ARS/2020-09-08`?

Reply:
Yes

Reply:
Mmm, and what do you think in something more representative? 

* `https://apis.fromdoppler.com/currency/conversion-value/ARS/2020-09-08?` 
* `https://apis.fromdoppler.com/currency/USD/ARS/2020-09-08?` 
* `https://apis.fromdoppler.com/currency/USD-value/ARS/2020-09-08?`

Reply:
Yes let me check these options, I like `conversion-value` or only `conversion`, I'll change it

Reply:
Done